### ISSUE TITLE
Turkish Names

### ISSUE THREAD

**Issue by emresaracoglu** [DavideViolante/gender-detection-from-name#89]
Hello, 

Do you plan to add Turkish names in future versions?

Thank you.

**Comment by emresaracoglu** [comment]
There are 200+ Turkish names for the first step. I created this with ChatGPT but each name is checked.

```
const TurkishNames = new Map([
  ['radife', 'female'],
  ['emre', 'male'],
  ['berkay', 'male'],
  ['sıddık', 'male'],
  ['ayşe', 'female'],
  ['mehmet', 'male'],
  ['elif', 'female'],
  ['ali', 'male'],
  ['fatma', 'female'],
  ['ahmet', 'male'],
  ['zehra', 'female'],
  ['murat', 'male'],
  ['sevgi', 'female'],
  ['mustafa', 'male'],
  ['büşra', 'female'],
  ['burak', 'male'],
  ['leyla', 'female'],
  ['tuncay', 'male'],
  ['dilek', 'female'],
  ['hakan', 'male'],
  ['sema', 'female'],
  ['selim', 'male'],
  ['aysel', 'female'],
  ['gökhan', 'male'], // değiştirildi
  ['yıldız', 'female'],
  ['yusuf', 'male'],
  ['ece', 'female'],
  ['onur', 'male'], // değiştirildi
  ['nazlı', 'female'],
  ['tahir', 'male'],
  ['özlem', 'female'],
  ['serkan', 'male'],
  ['feride', 'female'],
  ['sinan', 'male'],
  ['nazan', 'female'],
  ['ömer', 'male'],
  ['hatice', 'female'],
  ['eren', 'male'],
  ['banu', 'female'],
  ['kenan', 'male'],
  ['melike', 'female'],
  ['umit', 'male'],
  ['didem', 'female'],
  ['oğuz', 'male'],
  ['tuba', 'female'],
  ['engin', 'male'],
  ['nazlıcan', 'female'],
  ['cenk', 'male'], // değiştirildi
  ['şule', 'female'],
  ['cansu', 'female'],
  ['selçuk', 'male'],
  ['azra', 'female'],
  ['ismail', 'male'],
  ['nursel', 'female'],
  ['doğan', 'male'], // değiştirildi
  ['derya', 'female'],
  ['taylan', 'male'],
  ['merve', 'female'],
  ['kadir', 'male'],
  ['nermin', 'female'],
  ['koray', 'male'],
  ['aslı', 'female'],
  ['furkan', 'male'],
  ['nur', 'female'],
  ['sezgin', 'male'],
  ['pınar', 'female'],
  ['rahmi', 'male'],
  ['emine', 'female'],
  ['volkan', 'male'],
  ['aysun', 'female'], // değiştirildi
  ['fırat', 'male'],
  ['gülay', 'female'],
  ['atakan', 'male'],
  ['sibel', 'female'],
  ['muhammed', 'male'],
  ['zeynep', 'female'],
  ['şaban', 'male'],
  ['bilge', 'female'], // değiştirildi
  ['çağatay', 'male'],
  ['derin', 'female'],
  ['samet', 'male'],
  ['gizem', 'female'],
  ['hayriye', 'female'],
  ['gülsüm', 'female'], // değiştirildi
  ['ramazan', 'male'],
  ['sevda', 'female'],
  ['yasin', 'male'],
  ['hilal', 'female'],
  ['özkan', 'male'],
  ['hanife', 'female'],
  ['taner', 'male'],
  ['çiğdem', 'female'], // değiştirildi
  ['orhan', 'male'],
  ['nevin', 'female'], // değiştirildi
  ['barış', 'male'],
  ['yasemin', 'female'], // değiştirildi
  ['süleyman', 'male'],
  ['kıymet', 'female'],
  ['ayhan', 'male'],
  ['kübra', 'female'],
  ['hasan', 'male'], // değiştirildi
  ['mine', 'female'], // değiştirildi
  ['ferhat', 'male'],
  ['evrim', 'female'],
  ['mert', 'male'],
  ['müge', 'female'],
  ['ilker', 'male'],
  ['asya', 'female'],
  ['recep', 'male'],
  ['ışıl', 'female'],
  ['çetin', 'male'],
  ['serap', 'female'],
  ['muharrem', 'male'],
  ['necla', 'female'],
  ['ismet', 'male'],
  ['bahar', 'female'], // değiştirildi
  ['erdal', 'male'],
  ['füsun', 'female'],
  ['birol', 'male'],
  ['aysenur', 'female'],
  ['fikret', 'male'],
  ['ceren', 'female'],
  ['tarık', 'male'],
  ['gülşen', 'female'],
  ['ufuk', 'male'],
  ['pelin', 'female'],
  ['ayberk', 'male'],
  ['aylin', 'female'],
  ['levent', 'male'],
  ['ebru', 'female'],
  ['sedat', 'male'],
  ['tuğçe', 'female'],
  ['harun', 'male'],
  ['cemre', 'female'],
  ['rıza', 'male'],
  ['sabri', 'male'],
  ['esra', 'female'],
  ['kemal', 'male'],
  ['esin', 'female'],
  ['gül', 'female'],
  ['orçun', 'male'],
  ['nuriye', 'female'],
  ['alper', 'male'],
  ['semiha', 'female'],
  ['ercan', 'male'],
  ['buket', 'female'],
  ['gökçen', 'female'],
  ['erdem', 'male'],
  ['çisem', 'female'],
  ['şener', 'male'],
  ['belgin', 'female'],
  ['haydar', 'male'],
  ['vildan', 'female'],
  ['tanju', 'male'],
  ['aynur', 'female'],
  ['bilal', 'male'],
  ['müzeyyen', 'female'],
  ['sefa', 'male'],
  ['gülten', 'female'],
  ['zeki', 'male'],
  ['pervin', 'female'],
  ['ibrahim', 'male'],
  ['fatih', 'male'],
  ['cemil', 'male'],
  ['sevgül', 'female'],
  ['özcan', 'male'],
  ['doğan', 'male'], // değiştirildi
  ['ruhiye', 'female'],
  ['merih', 'male'],
  ['arzu', 'female'],
  ['baki', 'male'],
  ['semra', 'female'],
  ['mesut', 'male'],
  ['mualla', 'female'],
  ['ayfer', 'female'],
  ['cengiz', 'male'],
  ['burçin', 'female'],
  ['cavit', 'male'],
  ['binnaz', 'female'],
  ['lale', 'female'],
  ['umut', 'male'],
  ['saniye', 'female'],
  ['çağrı', 'male'],
  ['ziynet', 'female'],
  ['rıdvan', 'male'],
  ['ayten', 'female'],
  ['perihan', 'female'],
  ['şamil', 'male'],
  ['tamer', 'male'],
  ['şimay', 'female'],
  ['erdinç', 'male'],
  ['funda', 'female'],
  ['salih', 'male'],
  ['arife', 'female'],
  ['cem', 'male'],
  ['vedat', 'male'],
  ['kamil', 'male'],
  ['inci', 'female'],
  ['tuncer', 'male'],
  ['hulusi', 'male'],
  ['şermin', 'female'],
  ['kamuran', 'male'],
  ['muazzez', 'female'],
  ['bekir', 'male'],
  ['tevhide', 'female'],
  ['tevfik', 'male'],
  ['belma', 'female'],
  ['erdoğan', 'male'],
  ['feyza', 'female'],
  ['hayati', 'male'],
  ['beliz', 'female'],
  ['adnan', 'male'],
  ['elifnur', 'female'],
  ['metin', 'male'],
  ['neşe', 'female'],
  ['fahri', 'male'],
  ['aydın', 'male'],
  ['esma', 'female'],
  ['feyyaz', 'male'],
  ['melda', 'female'],
  ['yiğit', 'male'],
  ['tuana', 'female'],
  ['tayfun', 'male']
]);
```

**Comment by DavideViolante** [comment]
Thanks, that's a good start, do you want to add by yourself with a PR, or you want I do the changes?

**Comment by DavideViolante** [comment]
Resolved in https://github.com/DavideViolante/gender-detection-from-name/commit/d70829996bd8ec00e0b0a6e6936a558ed6c7fbd8
Available in version v1.11.0
